ui/docs/reference/core/hooks/exchanges.md

```diff
@@ -0,0 +1,9 @@
+---
+order: 2
+inline: true
+title: events
```

Comment:
The title should probably be `exchanges`.

Reply:
Good point!  I'm impressed you spotted that in a 27,000 line PR :)